app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -1,71 +1,89 @@
-- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+The address of your teacher training provider may not be where you spend most of your time. For example, your teacher training provider might be a university, but you’ll spend most of your time in placement schools.
 
-You should check the fees for a particular course you are interested in with the course provider.
+## Decide if you want a PGCE or PGDE
 
-## Earn a salary while you train
-These courses are usually only suitable for specific candidates, for example, some are for people who already have relevant teaching experience or are already working as unqualified teachers. You will be able to find out more about eligibility from the training provider. You may be able to study part-time.
+All primary and secondary postgraduate teacher training courses lead to QTS. Some also offer a [postgraduate certificate in education (PGCE) or a postgraduate diploma in education (PGDE)](/what-is-a-pgce).
 
-### School Direct (salaried)
+You do not need a PGCE or PGDE to become a teacher. 
 
-If you have the right experience, School Direct (salaried) give you QTS with many courses also offering a PGCE or PGDE. It takes a year to complete.
+It can be helpful to have a PGCE or PGDE if you plan on doing a master’s degree because it gives you credits towards one. 
 
-[Find postgraduate training to be a teacher, including School Direct (salaried) courses](https://www.gov.uk/find-postgraduate-teacher-training-courses)
+## Fund your training
 
-### Teach First
-Teach First is a 2-year programme leading to a PGDE and QTS. At the end of your first year, you will have QTS. The PGDE is awarded at the end of the second year.
+Courses usually cost £9,250 or more for [international candidates](/non-uk-teachers). 
 
-[Find out more about Teach First](https://www.teachfirst.org.uk/)
+If you train to teach certain secondary subjects, you may be eligible for a [scholarship or bursary](/funding-your-training#bursaries-and-scholarships).
 
-### Postgraduate teaching apprenticeship
+No matter what course you choose, you can apply for a student loan.
 
-Apprenticeships offer a year of school-based training which gives you QTS. Some also offer a PGCE or equivalent qualification. 
+[Learn about funding](/funding-your-training).
 
-[Find salaried teacher training including apprenticeships](https://www.find-postgraduate-teacher-training.service.gov.uk/)  by applying the 'only show courses that come with a salary' filter.
\ No newline at end of file
+## Check your eligibility 
+
+Your degree grade can affect your eligibility for courses. Teacher training providers often ask for a 2:2, but some ask for a third or pass.
+
+If you need help understanding degree grades, [chat to us online](/train-to-be-a-teacher/if-you-have-a-degree#talk-to-us). 
+
+### Find out if you're eligible for a salary  
```

Comment:
```suggestion
### Find out if you're eligible for a salary  
```
Should we say '... salary while you train' or '... a salaried course' to make it clear this relates to training rather than teaching?!